packages/rmf-auth/lib/utils/url.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
If this is the case `rmf-auth` will depend on `react-components`. And dashboard will depend on `react-components` and `rmf-auth`. Maybe an option, but it may be something overkill is to have our own common-utils package

Reply:
Hmm yea, the current dumb dependencies system probably can't handle transitive dependencies. Maybe we should start looking at moving to npm@7 with the new workspaces feature, or move to yarn, or use one of the meta repo utils like lerna.

For now I guess duplicating the code is fine, I agree that a common-utils package might be abit overkill if this is the only code in there.